tests/Rule/LatteTemplatesRule/EngineBootstrap/engine.bootstrap.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Latte\Engine;
+
+$engine = new Engine();
+$engine->addFilter('closure', function () {
```

Comment:
will this check closure parameters? for example here there is no parameter in function() but $title is given but no error is reported

Reply:
Hmm just checked it and for some reson it is always build just like \Closure without types. I will investigate why